css/style.css

```diff
@@ -0,0 +1,302 @@
+body {
+  font-family: "Montserrat", sans-serif;
+  
+}
+
+h1 {
+  font-size: 32px;
+}
+
+h2 {
+  font-size: 28px;
+
+
+}
+
+h3 {
+  font-size: 25px;
+}
+
+h5 {
+  font-size: 16px;
+}
+
+p {
+  font-size: 16px;
+  font-weight: 100;
+}
+
+.btn {
+  padding: 0.5rem 1.5rem;
+  border: none;
+  border-radius: 5px;
+  color: white;
+  border: none;
+}
+
+.section__background--one {
```

Comment:
Jeśli mamy BEM i modyfikator tj. `section__background--one` to powinien też występować element `section__background` - to on ma cechy wspólne dla elementów, a modyfikator szczególne.